docs/migration.md

```diff
@@ -0,0 +1,57 @@
+# Migration
+
+Existing servers can migrate from one database backend to another. This page includes guidance for migrating from sqlite - similar concepts apply when migrating from databases of other types.
```

Comment:
Nit, here and below: proper casing for the projects: SQLite, LLDAP, PostrgeSQL, ...